extensions/amp-a4a/0.1/test/testdata/valid_css_at_rules_amp.reserialized.js

```diff
@@ -26,6 +26,59 @@ export const data = {
 }
 </script></body></html>`,
 
+  minifiedCreative: '<!doctype html><html ⚡4ads><head><meta charset=utf-8><meta content=width=device-width,minimum-scale=1 name=viewport><link href=https://fonts.googleapis.com/css?family=Questrial rel=stylesheet type=text/css><style amp-custom>\n    amp-user-notification.amp-active {\n      opacity: 0;\n    }\n  </style><style amp4ads-boilerplate>body{visibility:hidden}</style></head><body>Hello, world.\n\n</body></html>',
```

Comment:
What does this file do?

Reply:
This just holds test data that we use for various amp-a4a tests.